opal/datatype/opal_convertor.c

```diff
@@ -45,11 +45,14 @@
 static void *opal_convertor_accelerator_memcpy(void *dest, const void *src, size_t size, opal_convertor_t *convertor)
 {
     int res, dev_id;
+    int tmp = -1;
+    uint64_t flags = 0;
     if (!(convertor->flags & CONVERTOR_ACCELERATOR)) {
         return MEMCPY(dest, src, size);
     }
 
-    if (NULL != opal_accelerator.memcpy) {
+    if ((opal_accelerator.check_addr(dest, &tmp, &flags) > 0) ||
```

Comment:
If we're going to use these checks anyway, it's probably worth setting the DtoH or DtoD, etc. flags. The current cuda impl doesn't utilize them, but if we're already doing these checks, it makes sense to pass the information through. Same comment for similar checks

Reply:
removed commit